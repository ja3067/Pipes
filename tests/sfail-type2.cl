def foo(x : float, y : float):
	return x + y

x = 4
y = 4.0

foo(x, y)

