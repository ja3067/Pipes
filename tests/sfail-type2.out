STypeError: expression of type int cannot be assigned to variable 'x' with explicit type float
